util/delete_entities.py

```diff
@@ -27,10 +28,17 @@ def parse_args():
     return parser.parse_args()
 
 
-def delete_entities(entity_type, entities):
-    fapi.delete_entity_type(args.workspace_namespace, args.workspace_name,
-                            entity_type, entities)
-    fapi._check_response_code(resp, 200)
+def _chunk_list(l, size):
+    return [
+        l[i:i + size] for i in xrange(0, len(l), size)
+    ]
+
+
+def delete_entities(args, entity_type, entities):
+    for chunked in _chunk_list(entities, ENTITY_CHUNK_SIZE):
+        resp = fapi.delete_entity_type(args.workspace_namespace, args.workspace_name,
```

Comment:
Print something like "Deleting chunk 0 for sample entities", otherwise the user is wondering what's going on

Reply:
Done.